includes/blocks/safe-svg/edit.js

```diff
@@ -48,7 +48,7 @@ const SafeSvgBlockEdit = ( props ) => {
 	} = attributes;
 	const blockProps = useBlockProps(
 		{
-			className:` safe-svg-cover`,
+			className:` wp-block-safe-svg-svg-icon`,
```

Comment:
Probably edge case but there may be developers that have added specific styling based on the `safe-svg-cover` class, so would probably be best to keep that class along with this new one I think
```suggestion
			className:` wp-block-safe-svg-svg-icon safe-svg-cover`,
```

Reply:
That's a valid point. I have updated this.